test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -373,36 +375,40 @@ def test_framework_and_neuron_sdk_version(neuron):
     """
     image = neuron
 
-    tested_framework, neuron_tag_framework_version = get_neuron_framework_and_version_from_tag(
-        image
-    )
+    tested_framework, tag_framework_version = get_framework_and_version_from_tag(image)
```

Comment:
can we remove `get_neuron_framework_and_version_from_tag` if not used anywhere else?

Reply:
Already removed, it's been replaced by get_neuron_release_manifest